src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -342,6 +347,19 @@ This is done using a [platform channel][],
 or through the interfaces defined in a platform
 interface package.
 
+### Add support to new platforms in an existing plugin project
+
+To support new platforms in an existing plugin project, run `flutter create` with
+the `--template=plugin` flag again in the project directory.
+For example, to add web support in an existing plugin, run:
+
+```
```

Comment:
\`\`\` => ```sh

Reply:
I saw above examples used "terminal", i'll use "terminal" to match.